brake/backends/cachebe.py

```diff
@@ -51,25 +52,18 @@ def _keys(self, func_name, request, ip=True, field=None, period=None):
 
     def count(self, func_name, request, ip=True, field=None, period=60):
         """Increment counters for all relevant cache_keys given a request."""
-        cache_keys = self._keys(func_name, request, ip, field, period)
-        counters = dict((key, 1) for key in cache_keys)
-        counters_read = cache.get_many(cache_keys)
-        for key, value in counters_read.items():
-            #Handle old values.
+        counters = dict((key, (1, time.time() + period)) for key in self._keys(
+            func_name, request, ip, field, period))
+        counters.update(cache.get_many(counters.keys()))
+        for key, value in counters.items():
+            # Handle old values.
             if isinstance(value, tuple):
-                count, _ = value # Futureproofing for expiration values.
+                count, expiration = value
             else:
                 count = value
-
+                expiration = time.time()
```

Comment:
It'd probably be a negative TTL, which would indeed be treated as immediately expired. We should also guard against setting the timeout to 0, because that would mean that key doesn't ever TTL out.